lib/ui/playground/styles/stage_card_styles.dart

```diff
@@ -0,0 +1,103 @@
+import 'package:farmsmart_flutter/ui/common/stage_card.dart';
+import 'package:flutter/widgets.dart';
+
+class StageCardStyles {
+  static StageCardStyle getUpcomingStageStyle() {
```

Comment:
👍

Reply:
let´s not use get when it´s not a true getter, maybe build? or just the name with no prefix?